optuna/notebooks/jupytercon_demo/RAPIDS_xfeat_Optuna-GPU.ipynb

```diff
@@ -0,0 +1,1803 @@
+       "3              9      548.0      28    11  ...                  270.0   \n",
+       "4              4      840.0      82    14  ...                  589.0   \n",
+       "\n",
+       "   DistanceDest_TE_plus  DistanceOrigin_TE_plus  \\\n",
+       "0            927.190614              927.136259   \n",
+       "1            334.246771              334.116294   \n",
+       "2           1195.190434             1195.217554   \n",
+       "3            270.140496              270.122800   \n",
+       "4            589.156523              589.199485   \n",
+       "\n",
+       "   DistanceUniqueCarrier_TE_plus  DivertedDest_TE_plus  \\\n",
+       "0                     927.171632              0.190614   \n",
+       "1                     334.188521              0.246771   \n",
+       "2                    1195.236261              0.190434   \n",
+       "3                     270.172147              0.140496   \n",
+       "4                     589.187968              0.156523   \n",
+       "\n",
+       "   DivertedOrigin_TE_plus  DivertedUniqueCarrier_TE_plus  \\\n",
+       "0                0.136259                       0.171632   \n",
+       "1                0.116294                       0.188521   \n",
+       "2                0.217554                       0.236261   \n",
+       "3                0.122800                       0.172147   \n",
+       "4                0.199485                       0.187968   \n",
+       "\n",
+       "   Dest_TEOrigin_TE_plus  Dest_TEUniqueCarrier_TE_plus  \\\n",
+       "0               0.326873                      0.362246   \n",
+       "1               0.363066                      0.435292   \n",
+       "2               0.407988                      0.426696   \n",
+       "3               0.263295                      0.312642   \n",
+       "4               0.356008                      0.344492   \n",
+       "\n",
+       "   Origin_TEUniqueCarrier_TE_plus  \n",
+       "0                        0.307891  \n",
+       "1                        0.304815  \n",
+       "2                        0.453816  \n",
+       "3                        0.294947  \n",
+       "4                        0.387454  \n",
+       "\n",
+       "[5 rows x 121 columns]"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# We cast to objects for categorical  and target encoding\n",
+    "# Can't pass categorical directly to LR\n",
+    "for col in CAT_COLS:\n",
+    "    df_[col] = df_[col].astype(\"object\")\n",
+    "\n",
+    "with timed(\"Feature Engineering\"):\n",
+    "    df_feature_eng = feature_engineering(df_)\n",
+    "    df_feature_eng[TARGET_COL] = df_feature_eng[TARGET_COL].astype('float64')\n",
+    "    score = train_and_eval(df_feature_eng)\n",
+    "    print(\"After feature eng: \", score)\n",
+    "\n",
+    "df_feature_eng.head()"
```

Comment:
Maybe add a little commentary? Like - "You can see additive features with names ending in "_plus" generated by xfeat and note that our 20(??) column dataset has grown to XX(?) columns with all of the generated features."